docs/001_develop/03_client-capabilities/009_presentation/avatar.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Avatar'
+sidebar_label: 'Avatar'
+id: client-presentation-avatar
+keywords: [presentation, avatar]
+tags:
+- presentation
+- avatar
+sidebar_position: 9
+---
+
+import AvatarDemo from './examples/avatar.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Avatar
+
+Use cases:
+* Display an image or text (usually initials) to represent a user.
+
+## Example
+
+<AvatarDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-avatar></rapid-avatar>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+        <rapid-avatar></rapid-avatar>
```

Comment:
Maybe add some set attributes here, just as this component is so simple. Same for the react and angular examples